src/modules/dashboard/components/Extensions/ExtensionActionButton.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useCallback } from 'react';
+import { defineMessages } from 'react-intl';
+import { useRouteMatch, useHistory, useParams } from 'react-router';
+
+import Button, { ActionButton, IconButton } from '~core/Button';
+import { ColonyExtensionQuery } from '~data/index';
+import { ExtensionData } from '~data/staticData/extensionData';
+import { ActionTypes } from '~redux/index';
+import { Address } from '~types/index';
+import { COLONY_EXTENSION_SETUP_ROUTE } from '~routes/index';
+
+const MSG = defineMessages({
+  enable: {
+    id: 'Extensions.ExtensionActionButton.enable',
+    defaultMessage: 'Enable',
+  },
+  install: {
+    id: 'Extensions.ExtensionActionButton.install',
+    defaultMessage: 'Install',
```

Comment:
Shouldn't this say "Add" rather than "Install"?

![Screenshot from 2020-11-17 14-26-20](https://user-images.githubusercontent.com/1193222/99390190-e1bd5880-28e0-11eb-95b1-cac416c49a88.png)